elements/pfe-jump-links/src/pfe-jump-links.js

```diff
@@ -203,8 +205,11 @@ class PfeJumpLinksNav extends PFElement {
         ${buildLinkList()}
     `;
     this.shadowRoot.querySelector("#container").innerHTML = html;
+
     let heading = document.createElement("h3");
-    heading.innerHTML = "Jump to section";
+    heading.innerHTML =
```

Comment:
Is this separate from the code on line 100?  It looks similar.

Reply:
Yep. One adresses the heading (shown in tablet+) and the other makes sure there is innerHTML in the button inside accordion.

Reply:
See my note above about having a "source of truth" for this `Jump to section` content.